_data/people.yml

```diff
@@ -290,6 +267,32 @@ zeeshan:
     display_name: "Zeeshan Ahmad"
     role: alumni-phd
     bio: Postdoctoral Research Scholar at University of Chicago
+    webpage: https://www.andrew.cmu.edu/user/azeeshan/
```

Comment:
bio: Postdoctoral Scholar at University of Chicago
webpage: https://ahzeeshan.github.io
email: azeeshan@alumni.cmu.edu

Reply:
Changes suggested in comments